code/modules/clothing/under/miscellaneous.dm

```diff
@@ -88,43 +88,39 @@
 	icon_state = "solgovr"
 	item_color = "solgovr"
 
-/obj/item/clothing/under/rank/centcom_officer
-	desc = "It's a jumpsuit worn by CentComm Officers."
-	name = "\improper CentComm officer's jumpsuit"
-	icon_state = "officer"
-	item_state = "g_suit"
-	item_color = "officer"
-
-/obj/item/clothing/under/rank/centcom_commander
-	desc = "It's a jumpsuit worn by CentComm's highest-tier Commanders."
-	name = "\improper CentComm officer's jumpsuit"
-	icon_state = "centcom"
-	item_state = "dg_suit"
-	item_color = "centcom"
+/obj/item/clothing/under/rank/centcom/ranker
+	name = "\improper Nanotrasen Officers Uniform"
+	desc = "Gold and green trim on authoritarian sage cloth, this uniform displays the rank of \"Ensign\" and bears \"N.A.S. Trurl\" on the left shoulder."
+	icon_state = "centcomm_ranker"
+	item_state = "ro_suit"
+	item_color = "centcomm_ranker"
+	displays_id = 0
+	flags_size = ONESIZEFITSALL
 
 /obj/item/clothing/under/rank/centcom/officer
-	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant-Commander\" and bears \"N.C.V. Fearless CV-286\" on the left shoulder."
-	name = "\improper Nanotrasen Officers Uniform"
-	icon_state = "officer"
-	item_state = "g_suit"
-	item_color = "officer"
+	name = "\improper Nanotrasen Commanders Uniform"

```

Comment:
"Commander's". As in, it belongs to the commander. Or perhaps "Commanders'" as in it belongs to the commanders, plural. I think. Maybe I'm wrong. Also, more generally there is no such rank as "Nanotrasen Commander" to the best of my knowledge.